src/content/en/fundamentals/native-hardware/recording-video/index.md

```diff
@@ -0,0 +1,212 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Most browsers can get access to the user's camera.
+
+{# wf_updated_on: 2017-09-24 #}
+{# wf_published_on: 2017-09-24 #}
+{# wf_blink_components: Blink>GetUserMedia #}
+
+# Recording Video from the User {: .page-title }
+
+{% include "web/_shared/contributors/mscales.html" %}
+
+Many browsers now have the ability to access video and audio input from the
+user. However, depending on the browser it might be a full dynamic and inline
+experience, or it could be delegated to another app on the user's device.
+
+## Start simple and progressively
+
+The easiest thing to do is simply ask the user for a pre-recorded
+file. Do this by creating a simple file input element and adding
+an `accept` filter that indicates we can only accept video files and ideally we
+will get them directly from the camera.
+
+    <input type="file" accept="video/*" capture="camera">
```

Comment:
Does this include the iOS changes?

Reply:
Good catch. It turns out that `capture="camera"` and `capture="microphone"` were both actually removed from the spec 5 years ago. I'll get rid from all three articles and mention the newer version.